app/org/sagebionetworks/bridge/validators/StudyValidator.java

```diff
@@ -79,6 +84,7 @@ public void validate(Object obj, Errors errors) {
         validateEmails(errors, study.getSupportEmail(), "supportEmail");
         validateEmails(errors, study.getTechnicalEmail(), "technicalEmail");
         validateEmails(errors, study.getConsentNotificationEmail(), "consentNotificationEmail");
+        validateDataGroupNamesAndFitForSynapseExport(errors, study.getDataGroups());
```

Comment:
They seem to be unrestricted. An example: 7-APHSpatialSpanMemory-4A04F3D0-AC05-11E4-AB27-0800200C9A66.